DataDog/storedog
# Update docker-compose.yml to use a build context for postgres service

## Description

Add new postgres custom image to be consumed by ECS

## How to test

1. Pull down branch
2. Use local build context to verify DB stands up correctly
